mail_activity_team/models/mail_activity.py

```diff
@@ -20,8 +20,10 @@ def _get_default_team_id(self, user_id=None):
             )
         return self.env["mail.activity.team"].search(domain, limit=1)
 
-    user_id = fields.Many2one(required=False)
-    team_user_id = fields.Many2one(related="user_id", readonly=False)
+    user_id = fields.Many2one(string="user", required=False)
+    team_user_id = fields.Many2one(
+        string="team user", related="user_id", readonly=False
+    )
```

Comment:
```suggestion
    user_id = fields.Many2one(string="User", required=False)
    team_user_id = fields.Many2one(
        string="Team user", related="user_id", readonly=False
    )
```

Reply:
@pedrobaeza , I have updated as per your feedback. Kindly check it.